.github/workflows/ci.yml

```diff
@@ -169,6 +169,14 @@ jobs:
       if: ${{ matrix.db != 'sqlite' }}
     - name: pnpm install
       run: pnpm install --offline 
+    - name: Linting for test suite sql-mapper
+      run: cd packages/sql-mapper && npx standard; cd ../..
+    - name: Linting for test suite sql-json-schema-mapper
+      run: cd packages/sql-json-schema-mapper && npx standard; cd ../..
+    - name: Linting for test suite sql-openapi
+      run: cd packages/sql-openapi && npx standard; cd ../..
+    - name: Linting for test suite sql-graphql
+      run: cd packages/sql-graphql && npx standard; cd ../..
```

Comment:
That's going to add quite a bit of duplication between the CI runs. Create a new job with just these and without starting a database.

Reply:
Oh yes, I have seen... I will create a new dedicated job for linting, I'm learning about ci configuration.
Thank you.

Reply:
Ok, I have split `ci-db-core` job into `ci-db-core-lint` and `ci-db-core-test` jobs.
`ci-db-core-test` replace the old job name `ci-db-core.

It's ok for you?